app/views/templates/package_action.erb

```diff
@@ -0,0 +1,46 @@
+<%#
+kind: job_template
+name: Package Action - SSH Default
+job_name: PackageAction
+provider_type: Ssh
+template_inputs:
+- name: pre_script
+  description: A script to run prior to the package action
+  input_type: user
+  required: false
+- name: action
+  description: 'The package action: install, update, or remove'
+  input_type: user
+  required: true
+  options: "install\nupdate\nremove"
+- name: package
+  description: The name of the package, if any
+  input_type: user
+  required: false
+- name: post_script
+  description: A script to run after the package action
+  input_type: user
+  required: false
+%>
+
+# Pre Script
+<%= input("pre_script") %>
+PRE_RETVAL=$?
+
+# Action
+<% if @host.operatingsystem.family == 'Redhat' %>
+yum -y <%= input("action") %> <%= input("package") %>
+<% elsif @host.operatingsystem.family == 'Debian' %>
+apt-get -y <%= input("action") %> <%= input("package") %>
+<% end %>
```

Comment:
For the katello case (without provisioning) this would require the OS to be created and set for each system).  I do think this makes sense but isn't something i had considered before.  Will make a note for host unification.